WebGoatCore/Controllers/CheckoutController.cs

```diff
@@ -28,6 +28,7 @@ public CheckoutController(UserManager<IdentityUser> userManager, CustomerReposit
             _shipperRepository = shipperRepository;
             _orderRepository = orderRepository;
             _resourcePath = configuration.GetValue(Constants.WEBGOAT_ROOT, hostEnvironment.ContentRootPath);
+            _model = new CheckoutViewModel();
```

Comment:
This won't work, look at Checkout() method, it does InitializeModel() if _model is null, now it won't be called.
And InitializeModel() uses HttpContext.Session, are we sure it will be available in constructor?